cypress/integration/appTest.js

```diff
@@ -0,0 +1,26 @@
+describe('App Home Page', () => {
+
+  beforeEach(() => {
+    cy.fixture('multipleCities.json').as('cities').then((cities) => {
+      cy.intercept('GET', 'https://softwhere-api.herokuapp.com/cities', {body: cities});
+    });
+
+    cy.visit('http://localhost:3000');
+  });
+
+  it('User should see a header containing the app title and description', () => {
+    cy.get('header').first().contains('softWhere?');
+    cy.get('header').last().contains('Best US Cities to Live as a Software Developer');
+  });
+
+  it('User should see all available city cards when the page loads', () => {
+    cy.get('.cities-container')
+    .children().should('have.length', 4);
```

Comment:
One of the things I really enjoyed about Cypress is the fact that there are seemingly endless ways to test.  I'd not seen the .children stub(? method ? 🤷) before, but I'm totally going to use it now!